core/client/app/mixins/editor-base-controller.js

```diff
@@ -211,12 +211,13 @@ export default Ember.Mixin.create({
 
     showSaveNotification: function (prevStatus, status, delay) {
         var message = this.messageMap.success.post[prevStatus][status],
-            path = this.get('model.absoluteUrl'),
             type = this.get('postOrPage'),
             notifications = this.get('notifications');
 
         if (status === 'published') {
-            message += `&nbsp;<a href="${path}">View ${type}</a>`;
+            message += `&nbsp;<a href="${this.get('model.absoluteUrl')}">View ${type}</a>`;
+        } else {
+            message += `&nbsp;<a href="${this.get('model.previewUrl')}">Preview ${type}</a>`;
```

Comment:
Totally! That then leads me to another question: it would be unintuitive if the same notification sent you to a new page sometimes but not other times (published vs draft). Should both, then, get the `_blank`?